app/main.py

```diff
@@ -1,2 +1,48 @@
+from typing import Any, Hashable
+
+
+class Node:
```

Comment:
This class better be defined inside of a Dictionary since it is a part of its implementation and will not be used outside of it. Also you should store hash value in it as well.

> Also, don't forget to store (key, hash, value) as a node in the hash table. A hash table may be stored as a simple list of nodes.